bpaotu/bpaotu/templates/bpaotu/otu_log.html

```diff
@@ -0,0 +1,35 @@
+<style>
```

Comment:
could you inherit from the base template, or at least add a <body> and <title> and so on?

Reply:
oh and add a heading

Reply:
Fixed.